sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -63,6 +63,8 @@ export enum AppRoutes {
   Reports = "reports",
   ApplicationExceptionsApproval = "application-exceptions/:exceptionId",
   Exceptions = "exceptions",
+  OfferingChangeRequests = "institution/offering/change-requests",
+  ViewOfferingChangeRequest = "institution/offering/view-change-request/:offeringId/program/:programId",
```

Comment:
`"institution/offering/:offeringId/program/:programId/view-change-request",`?

Reply:
I changed it to `institution/program/:programId/offering/:offeringId/view-change-request`